src/components/UploadEditor.vue

```diff
@@ -143,33 +141,43 @@ export default {
 @import '../assets/variables.scss';
 
 .upload-editor {
-	height: 100%;
 	&__previews {
-		display: flex;
-		flex-wrap: wrap;
+		overflow-x: hidden !important;
+		display: grid;
+		position: relative;
 		overflow: auto;
+		grid-template-columns: repeat(4, auto);
 	}
 	&__actions {
 		display: flex;
-		justify-content: space-between;
-		margin-bottom: 16px;
+		justify-content: flex-end;
 		margin-top: 16px;
+		margin-bottom: 4px;
+		button {
+			margin: 0 4px 0 4px;
+		}
 	}
 	&__add-more {
-		width: 100px;
-		height: 100px;
+		width: 80px;
+		height: 80px;
 		border: none;
-		border-radius: var(--border-radius-large);
+		border-radius: 40px;
```

Comment:
I think that the radius variable for full rounded borders is `var(--border-radius-pill)`.